wp-content/themes/chinapower/sass/pages/_archives.scss

```diff
@@ -241,26 +241,32 @@
 			}
 		}
 
-		.entry-header {
+		header.entry-header {
```

Comment:
I added ```header``` as the ```.entry-header``` styles on the archive page we're being overwritten by the ```.entry-header``` styles on the posts page [here](https://github.com/CSIS-iLab/chinapower_wp/blob/1f310d569909d3bf494fe2898e36ca64ee4d9292/wp-content/themes/chinapower/sass/pages/_posts.scss#L285)

Reply:
Got it. So my decision to add `article` & `header` & various other selectors is actually a bad thing - it results in some very hyper-specific CSS which is not what we want (and is what is causing the post page styles to overwrite the archive styles). Rather than adding `header.entry-header`, can you try removing `header` in both places? I'd be interested in knowing how that impacts things.